setup.py

```diff
@@ -41,6 +41,8 @@
             'graphviz',
             'tyto>=1.4',
             'openpyxl',
+            'certifi',
```

Comment:
I don't think `certify` needs to be included here. `requests` depends on it, but this utility does not depend directly on `certifi`. You can let `requests` bring in `certifi`, sbol-utilities doesn't have to.

Reply:
Thank you; you were right that deleting it did not hurt anything.